Utilities/XrdAdaptor/src/XrdStatistics.h

```diff
@@ -41,8 +46,16 @@ class XrdStatisticsService
 
     void fillDescriptions(edm::ConfigurationDescriptions &descriptions);
 
-private:
+    struct CondorIOStats {
+      uint64_t bytesRead{0};
+      std::chrono::nanoseconds transferTime{0};
+    };
 
+    // Provide an update of per-site transfer statistics to the CondorStatusService.
+    // Returns a mapping of "site name" to transfer statistics.  The "site name" is
+    // as self-identified by the Xrootd host; may not necessarily match up with the
+    // "CMS site name".
+    std::map<std::string, CondorIOStats> condorUpdate();
```

Comment:
A `std::vector<std::pair<std::string, CondorIOStats>` would be more memory efficient and probably faster to fill.